### PR TITLE
Add teleport:6 nightly Docker image

### PR DESCRIPTION
**Pull request by webvictim**
Missed this as part of the post-release checklist.

This will publish the following tags: `6`, `5` and `4.4`

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1327,13 +1327,10 @@ steps:
...
       # build major version images which are just teleport:x
-      CURRENT_VERSION_ROOT: v5
-      # normally we only support the current release plus the two previous releases, but we're temporarily adding an
-      # extra version here during the migration over from x.y -> x.y-1 support to x -> x-1 support as per RFD 12.
-      OLD_CURRENT_VERSION_ROOT: v5.1
-      OLD_PREVIOUS_VERSION_ONE_ROOT: v5.0
-      OLD_PREVIOUS_VERSION_TWO_ROOT: v4.4
-      OLD_PREVIOUS_VERSION_THREE_ROOT: v4.3
+      CURRENT_VERSION_ROOT: v6
+      OLD_CURRENT_VERSION_ROOT: v5
```
Comment (awly) [inline .drone.yml:1331]: should this be `PREVIOUS_VERSION_ONE_ROOT`?
and should we remove v4.3?

Comment (webvictim) [reply .drone.yml:1331]: Yes to both!

**Review by stevenGravy — APPROVED** [review]

**Review by awly — APPROVED** [review]

**Comment by webvictim** [comment]
@russjones Please bot approve.

**Review by russjones — APPROVED** [review]
Bot.

**Pull request merged**